boards/arm/mps2_an521/board.cmake

```diff
@@ -12,6 +12,14 @@ set(QEMU_FLAGS_${ARCH}
   )
 board_set_debugger_ifnset(qemu)
 
+set(QEMU_FLAGS_HOST_SERIAL
+  # To enable a host tty switch between serial and pty
+  #  -chardev serial,path=/dev/ttyS0,id=hostS0
+
+  -chardev pty,id=hostS0
+  -serial chardev:hostS0
```

Comment:
Hmm, are you sure that it doesn't require some additional rights on host to do that?
Shouldn't we disable it by default to avoid regressions among the current users?

Reply:
Hi @evgeniy-paltsev ,

> are you sure that it doesn't require some additional rights on host to do that?

I didn't experiment permissions problems. I believe that permissions for the pts-N are the same used in the console.
See my steps on a Debian 10:

```console
$ west build -t run
-- west build: running target run
[3/12] Linking C executable zephyr/zephyr_pre0.elf

[6/12] Linking C executable zephyr/zephyr_pre1.elf

[11/12] Linking C executable zephyr/zephyr.elf
Memory region         Used Size  Region Size  %age Used
           FLASH:       14512 B         4 MB      0.35%
            SRAM:        4128 B         4 MB      0.10%
           SRAM1:          0 GB         4 MB      0.00%
        IDT_LIST:          0 GB         2 KB      0.00%
[11/12] To exit from QEMU enter: 'CTRL+a, x'[QEMU] CPU: cortex-m33
char device redirected to /dev/pts/2 (label hostS0)
qemu-system-arm: warning: nic lan9118.0 has no peer
*** Booting Zephyr OS build v2.7.99-2720-g49520dfb4699  ***
Hello World! mps2_an521
```

As you can see,` char device redirected to /dev/pts/2 (label hostS0)` was create. To access that second console, for instance, `minicom -D /dev/pts/N`.

The original console is created by https://github.com/zephyrproject-rtos/zephyr/blob/b1d2bb0b47c8b35738e09d3e2045675ff13bbcd4/cmake/emu/qemu.cmake#L63-L64

In this case, I believe that if user should handle permissions for the second PTS it is probable that he needs do something for the original console too.

> Shouldn't we disable it by default to avoid regressions among the current users?

It is not clear to me what you are referring. Are you suggesting to drop below?

```c
&uart1 {
	status = "okay";
};
```